reconcile/utils/terrascript_aws_client.py

```diff
@@ -4047,6 +4050,24 @@ def init_values(self, spec: ExternalResourceSpec, init_tags: bool = True) -> dic
             if val is not None:
                 values[key] = val
 
+        if self.versions and not self.versions.get(
+            spec.provisioner_name, ""
+        ).startswith("3"):
+            if spec.provider == "rds":
+                if db_name := values.pop("name", None):
+                    values["db_name"] = db_name
```

Comment:
shouldn't we use `values["name"]` for provider v3*

Reply:
this block is conditional only for non v3*, right?

Reply:
ah missed the `not` sorry